app/views/paye/paye_income_tax_and_nics.scala.html

```diff
@@ -118,22 +118,24 @@ <h1 class="heading-xlarge">
     </table>
   }
 
-    <table class="paye-ats-data-table">
-        <thead>
-            <tr>
-                <th scope="col">@Messages("paye.ats.adjustments")</th>
-                <th scope="col">@Messages("generic.amount_pounds")</th>
-            </tr>
-        </thead>
-        <tbody>
-            @for(row <- viewModel.adjustments) {
-                <tr id="@row.label">
-                    <th scope="row">
-                        @Messages("paye.ats.adjustments." + row.label)
-                    </th>
-                    <td class="numeric">@includes.currency_amount(row.adjustmentAmount, true)</td>
+    @if(viewModel.adjustments.nonEmpty){
```

Comment:
Is this missing a test for the empty case?

Reply:
Agreed 👍 added